website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Follow the steps to upgrade the Appsmith community Kubernetes installation to a Business Edition.
+---
+
+# Kubernetes
+
+The Business Edition (BE) Helm charts installation includes support for Horizontal Pod Auto Scaling (HPA), which allows Appsmith pods to be automatically scaled based on current load. Additionally, this means that Appsmith pods are managed using a Kubernetes _deployment_ resource instead of a _stateful-set_ resource. To upgrade from the Community Edition (CE) to the Business Edition (BE) installed on Kubernetes, follow the steps listed below.
+
+## Export backup
+
+1. Open a shell into one of your current Appsmith pods:
+
+   ```bash
+   kubectl exec -it <pod> bash
+   ```
+
+2. Run the backup command:
+
+   ```bash
+   appsmithctl backup
+   ```
+
+3. Once the backup process is complete, the backup archive is available at a location like this:
+
+   ```
+   /appsmith-stacks/data/backup/appsmith-backup-2022-10-24T07-09-56.930Z.tar.gz
```

Comment:
remove path from code block

Reply:
These changes will happen once we have the component merged.

Reply:
ignore this comment. Closing conversation